### ISSUE TITLE
Tags in Frontmatter

### ISSUE THREAD

**Issue by 0kuki** [kctekn/obsidian-TagsRoutes#7]
I have a Vault where almost all the tags are stored in frontmatter (media files index kinda)

It seems this plugin considers only tags that are in the file body, is that intentional?

**Comment by kctekn** [comment]
Yes, it is intentional. The plugin is currently designed to manage tags at the paragraph level, focusing on "the ideas and concepts across various notes." It does not handle tags in the note's frontmatter, as those are typically properties for the entire note. Categorizing notes at the file level is not supported at the moment, but this feature might be added in later versions.

**Comment by kctekn** [comment]
Please check the latest version 1.0.7, which has just added the support to tags in frontmatter
